### PR TITLE
add unit test for issue #993

### PR DESCRIPTION
**Pull request by whoburg**
@bqpd, ready for review.

### CODE REVIEW

File: gpkit/tests/t_solution_array.py [hunk 1]
```diff
@@ -76,6 +76,17 @@ def test_units_sub(self):
             "1000N" in
             sol.table().replace(" ", "").replace("[", "").replace("]", ""))
 
+    def test_key_options(self):
+        # issue 993
+        x = Variable("x")
+        y = Variable("y")
+        with SignomialsEnabled():
+            m = Model(y, [y + 6*x >= 13 + x**2])
+        msol = m.localsolve(verbosity=0)
```
Comment (bqpd) [inline gpkit/tests/t_solution_array.py]: How about also testing `m.prog.gps[-1].solve()`?

**Comment by bqpd** [comment]
test models please

**Comment by bqpd** [comment]
test models please

**Comment by bqpd** [comment]
test models please

**Comment by bqpd** [comment]
test models please

**Comment by bqpd** [comment]
@whoburg, I'm ready to merge this if it passes model tests.

**Comment by whoburg** [comment]
I keep seeing issues with one of the model repos so I somewhat don't expect the model tests to pass

**Comment by bqpd** [comment]
That's because we were waiting on the cvxopt update. From now on I'll be strict and only allow models passing on master in that test.

**Comment by bqpd** [comment]
Augh I see. When did models tests break.

**Comment by bqpd** [comment]
test models please

**Comment by bqpd** [comment]
@galbramc, any way we could have jho_Push triggered by pushes to the 1682 branch of gpkit-models? A change on that branch was causing the jho tests to fail...

**Comment by bqpd** [comment]
Merging because this is like 3 lines and the Windows 10 test machines has been hanging for 45 minutes...

**Pull request merged**